nodejs/angular-npm/README.md

```diff
@@ -0,0 +1,17 @@
+# Node.js Sample app using Npm and a Angular framework
+
+## Building
+
+`pack build angular-sample --buildpack gcr.io/paketo-buildpacks/nodejs --env "NODE_RUN_SCRIPTS=build" --env "NODE_ENV=development"`
+
+## Running
+
+`docker run --interactive --init --tty --env PORT=8080 --publish 8080:8080 angular-sample`
+
+## Viewing
+
+`curl http://localhost:8080`
+
+### Note
+
+When containerizing angular apps, be sure to specify the host when you `ng serve` the appliation in `package.json` using the `--host <0.0.0.0>` flag.
```

Comment:
Could you please add a note to why `"NODE_ENV=development"` was set? (That we need the `ng` cli to start the app). Also please do the same for the vue app.

Reply:
adressed [angular-npm](https://github.com/paketo-buildpacks/samples/commit/b83941f4efe0ef69c3e58d9ee0a044fec5852059#diff-ca17ae890a947685d3d74dd883c8613dc28264008a4a58cdb95fd8952fc44d3bR15) and [vue-npm](https://github.com/paketo-buildpacks/samples/commit/b83941f4efe0ef69c3e58d9ee0a044fec5852059#diff-ca17ae890a947685d3d74dd883c8613dc28264008a4a58cdb95fd8952fc44d3bR15)